### PR TITLE
Fix: conference rooms were no longer hidden

### PR DESCRIPTION
**Pull request by dbkr**

### CODE REVIEW

File: src/MatrixTools.js [hunk 1]
```diff
@@ -26,30 +26,48 @@ module.exports = {
...
 
-    isDirectMessageRoom: function(room, me, ConferenceHandler, hideConferenceChans) {
+    /**
+     * If the room contains only two members including the logged-in user,
+     * return the other one. Otherwise, return null.
+     */
+    getOnlyOtherMember(room, me) {
+        const joinedMembers = room.getJoinedMembers();
+
+        if (joinedMembers.length === 2) {
```
Comment (richvdh) [inline src/MatrixTools.js]: hrmmm.... what will this do if you are invited to a room with two other members?

**Comment by richvdh** [comment]
lgtm modulo comment

File: src/MatrixTools.js (hunk 1 above)
Comment (dbkr) [inline src/MatrixTools.js]: Good point - should be easy to check

**Comment by dbkr** [comment]
Does this seem reasonable?

**Comment by richvdh** [comment]
> Does this seem reasonable?

I guess so. I'm slightly sad that the implementation of "getOnlyOtherMember" doesn't really match the name or the comment, but it's certainly fine for what you're using it for. I'm happy for it to stay like this.

**Pull request merged**